hls4ml/converters/keras/qkeras_layers.py

```diff
@@ -72,26 +75,38 @@ def parse_qactivation_layer(keras_layer, input_names, input_shapes, data_reader,
             activation_config['config']['integer'] = 2
         else:
             activation_config['class_name'] = 'unknown'
-    
+
     if activation_config['class_name'] not in supported_activations:
         raise Exception('Unsupported QKeras activation: {}'.format(activation_config['class_name']))
 
+    if activation_config['class_name'] == 'quantized_bits':
+        activation_config['class_name'] = 'linear'
 
     if activation_config['class_name'] == 'ternary_tanh':
         layer['class_name'] = 'TernaryTanh'
         layer['threshold'] = activation_config.get('config', {}).get('threshold', 0.33)
         if layer['threshold'] is None:
             layer['threshold'] = 0.33 # the default ternary tanh threshold for QKeras
+        layer['activation'] = 'ternary_tanh'
+    elif ((activation_config['class_name'] == 'quantized_sigmoid'
+          and not activation_config['config']['use_real_sigmoid'])
+          or (activation_config['class_name'] == 'quantized_tanh'
+          and not activation_config['config']['use_real_tanh'])):
```

Comment:
Since these options are not in a QKeras release yet, maybe we should access them with a `.get` returning a default value of `False` for robustness

Reply:
I just committed the fix.